packages/contracts-bedrock/contracts/universal/Semver.sol

```diff
@@ -35,4 +37,22 @@ contract Semver {
         MINOR_VERSION = _minor;
         PATCH_VERSION = _patch;
     }
+
+    /**
+     * @notice Returns the full semver contract version.
+     *
+     * @return Semver contract version as a string.
+     */
+    function version() public view returns (string memory) {
```

Comment:
Could we get rid of the other getters?

Reply:
I guess, any strong reason why?

Reply:
Less pollution of the jump table, each extra fn will add gas overhead. The other getters are only useful if something onchain is going to act based on it imo. If they are just human readable, then we can convert to uint8s which should save on codesize + only have 1 getter function so that we don't have to add 4 fns to each contract that inherits this

Reply:
Weakly agree with @tynes here. Approving now anyways.

Reply:
slight nit: isn't there `strings.concat`?